docs/examples/hairpin_fitting/hairpin_unfolding.rst

```diff
@@ -0,0 +1,289 @@
+    plt.xlabel('Distance ($\mu$m)')
+
+.. image:: selected_data.png
+
+Define the models
+^^^^^^^^^^^^^^^^^
+
+Define the model for the DNA with folded hairpin::
+
+    dna_handles_force = lk.ewlc_odijk_force("dna_handles")
+
+The model for DNA and the unfolded hairpin is composed by summing the model for the DNA handles and the model for the hairpin with distance as the dependent parameter::
+
+    dna_handles_and_hairpin_distance = lk.ewlc_odijk_distance("dna_handles") + lk.efjc_distance("dna_ss_hairpin")
+
+Invert the model for DNA and hairpin such that force becomes the dependent parameter::
+
+    dna_handles_and_hairpin_force = dna_handles_and_hairpin_distance.invert(interpolate=True, independent_min=0, independent_max=90)
+
+Add the models to the fit::
+
+    fit = lk.FdFit(dna_handles_force, dna_handles_and_hairpin_force)
+
+
+Fit the data
+^^^^^^^^^^^^
+
+For fitting, we can either fit all the data at once by adding all the selected data to the fit. Another option is incremental fitting, where the DNA handles are fitted first. 
+The fitted parameters for the DNA handles can then be used as an estimate for fitting the unfolding event. Below, we use incremental fitting.
+
+First, we add data for the DNA handles only::
+
+    fit[dna_handles_force].add_data("DNA handles",force_back_folded,distance_back_folded)
+
+Set parameter bounds and first guess for the parameters::
+
+    fit["dna_handles/Lp"].lower_bound = 30 # in nanometers
+    fit["dna_handles/Lp"].upper_bound = 70 # in nanometers
+    fit["dna_handles/Lp"].value = 50
+    fit["dna_handles/Lc"].value = 1.7
+    fit["dna_handles/St"].value = 1500
+
+Fit the data before unfolding::
+
+    fit.fit()
+
+Plot the result of the fit::
+
+    plt.figure()
+    fit[dna_handles_force].plot()
+    plt.xlabel(r'Tether length ($\mu$m)')
+    plt.ylabel('Force (pN)')
+
+.. image:: fit_handles.png
+
+Now, add the data after the unfolding event::
+
+    fit[dna_handles_and_hairpin_force].add_data("DNA handles + unfolded hairpin",force_back_unfolded,distance_back_unfolded)
+
+This time all the selected data are fitted and the values for the DNA handles from the first part of the fit are used as initial guess.
+Sometimes, when fitting many unfolding events, the fit does not converge well when all data are fitted at once. If that happens, you can fix parameters from the first fit, for example by setting
+`fit["dna_handles/Lc"].fixed = True`. For this particular data set it is not necessary to fix parameters and we only provide initial guesses and parameter bounds for the hairpin::
+
+    fit["dna_ss_hairpin/Lp"].lower_bound = 0.5 # in nanometers
+    fit["dna_ss_hairpin/Lp"].value = 1.5 # in nanometers
+    fit["dna_ss_hairpin/Lp"].upper_bound = 2.0 # in nanometers
+
+    fit["dna_ss_hairpin/Lc"].value = 0.02 # in microns
+    fit["dna_ss_hairpin/Lc"].lower_bound = 0.001 # in microns
+
+    fit["dna_ss_hairpin/St"].value = 500 # in pN
+    fit["dna_ss_hairpin/St"].upper_bound = 2000 # in pN
```

Comment:
I see your point, but don't see a clear solution. We measure up to 30 pN, which is too high for the regular WLC, without stretch modulus. What would you recommend? Fixing the stretch modulus?

Reply:
Yes, that sounds like the best approach. Instead of adding another datafile, I could say something like 'we determined the St to be 1500pN' (without fitting in this particular notebook) and then fix St for this fit.

Reply:
Seems reasonable to me.